Formula/aws-sso-util.rb

```diff
@@ -0,0 +1,114 @@
+    sha256 "b3a77dcc7d54a3725aa0a9b44e4a79142a013584cd0568750a9ee9ab6970538f"
+  end
+
+  resource "click" do
+    url "https://files.pythonhosted.org/packages/27/6f/be940c8b1f1d69daceeb0032fee6c34d7bd70e3e649ccac0951500b4720e/click-7.1.2.tar.gz"
+    sha256 "d2b5255c7c6349bc1bd1e59e08cd12acbbd63ce649f2588755783aa94dfb6b1a"
+  end
+
+  resource "jmespath" do
+    url "https://files.pythonhosted.org/packages/3c/56/3f325b1eef9791759784aa5046a8f6a1aff8f7c898a2e34506771d3b99d8/jmespath-0.10.0.tar.gz"
+    sha256 "b85d0567b8666149a93172712e68920734333c0ce7e89b78b3e987f71e5ed4f9"
+  end
+
+  resource "jsonschema" do
+    url "https://files.pythonhosted.org/packages/69/11/a69e2a3c01b324a77d3a7c0570faa372e8448b666300c4117a516f8b1212/jsonschema-3.2.0.tar.gz"
+    sha256 "c8a85b28d377cc7737e46e2d9f2b4f44ee3c0e1deac6bf46ddefc7187d30797a"
+  end
+
+  resource "pyrsistent" do
+    url "https://files.pythonhosted.org/packages/f4/d7/0fa558c4fb00f15aabc6d42d365fcca7a15fcc1091cd0f5784a14f390b7f/pyrsistent-0.18.0.tar.gz"
+    sha256 "773c781216f8c2900b42a7b638d5b517bb134ae1acbebe4d1e8f1f41ea60eb4b"
+  end
+
+  resource "python-dateutil" do
+    url "https://files.pythonhosted.org/packages/4c/c4/13b4776ea2d76c115c1d1b84579f3764ee6d57204f6be27119f13a61d0a9/python-dateutil-2.8.2.tar.gz"
+    sha256 "0123cacc1627ae19ddf3c27a5de5bd67ee4586fbdd6440d9748f8abb483d3e86"
+  end
+
+  resource "PyYAML" do
+    url "https://files.pythonhosted.org/packages/a0/a4/d63f2d7597e1a4b55aa3b4d6c5b029991d3b824b5bd331af8d4ab1ed687d/PyYAML-5.4.1.tar.gz"
+    sha256 "607774cbba28732bfa802b54baa7484215f530991055bb562efbed5b2f20a45e"
+  end
+
+  resource "s3transfer" do
+    url "https://files.pythonhosted.org/packages/88/ef/4d1b3f52ae20a7e72151fde5c9f254cd83f8a49047351f34006e517e1655/s3transfer-0.5.0.tar.gz"
+    sha256 "50ed823e1dc5868ad40c8dc92072f757aa0e653a192845c94a3b676f4a62da4c"
+  end
+
+  resource "urllib3" do
+    url "https://files.pythonhosted.org/packages/4f/5a/597ef5911cb8919efe4d86206aa8b2658616d676a7088f0825ca08bd7cb8/urllib3-1.26.6.tar.gz"
+    sha256 "f57b4c16c62fa2760b7e3d97c35b255512fb6b59a259730f36ba32ce9f8e342f"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    env = { "AWS_CONFIG_FILE" => "#{testpath}/config" }
+    cmd = "#{bin}/aws-sso-util configure profile invalid"\
+          " --sso-start-url https://example.com/start --sso-region eu-west-1" \
+          " --account-id 000000000000 --role-name InvalidRole" \
+          " --region eu-west-1 --non-interactive"
+
+    expected = [
+      "",
+      "[profile invalid]",
+      "sso_start_url = https://example.com/start",
+      "sso_region = eu-west-1",
+      "sso_account_id = 000000000000",
+      "sso_role_name = InvalidRole",
+      "region = eu-west-1",
+      "credential_process = aws-sso-util credential-process --profile invalid",
+    ]
+
+    auth_process = IO.popen(env, cmd)
+
+    assert_match "", auth_process.read
```

Comment:
```suggestion
    output = shell_output "AWS_CONFIG_FILE=#{testpath}/config #{cmd}"

    assert_match "something", output
```

Reply:
@SMillerDev In this case I expect the output to be empty because `aws-sso-util` doesn't print anything on stdout and creates only the config file.

Is there a better way to verify this behavior?

Reply:
`assert_empty`. `assert_match ""` means that it needs to include at least `""`, which will always succeed

Reply:
Thank you for the clarification, done 👍🏻